nerfstudio/configs/base_config.py

```diff
@@ -149,3 +149,5 @@ class ViewerConfig(PrintableConfig):
     skip_openrelay: bool = False
     """Avoid using openrelay to communicate with the viewer. Try disabling if you have trouble
     connecting to the viewer"""
+    codec: str = "H264"
```

Comment:
Can you make this `codec: Literal["H264", "VP8"]`, Then the options can be removed from the docstring